backend/api/api.template.yml

```diff
@@ -946,6 +946,9 @@ Resources:
       Policies:
         - AWSLambdaVPCAccessExecutionRole
         - AWSXrayWriteOnlyAccess
+        - DynamoDBReadPolicy:
+            TableName:
+              Fn::ImportValue: !Sub ${DeploymentName}-DynamoDB-DataTableName
```

Comment:
just to confirm, you've tested the changes that need these permissions changes?

Reply:
Yep we ran through a journey of creating an account + service and then updating the new service to check this worked